kore/src/Kore/Step/Simplification/TermLike.hs

```diff
@@ -155,10 +156,10 @@ simplifyToOr
     =>  Predicate variable
     ->  TermLike variable
     ->  simplifier (OrPattern variable)
-simplifyToOr term predicate =
+simplifyToOr predicate term =
```

Comment:
Why reverse the argument order of `simplifyToOr` and not `simplifyInternal`?

Reply:
I didn't change the type, I think the arguments had their names reversed and this fixes that.